src/api-service/__app__/onefuzzlib/user_credentials.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from uuid import UUID
+
+import azure.functions as func
+import jwt
+from onefuzztypes.enums import ErrorCode
+from onefuzztypes.models import Error, Result, UserInfo
+
+
+def parse_jwt_token(request: func.HttpRequest) -> Result[UserInfo]:
+    """ Obtains the Access Token from the Authorization Header """
+
+    auth: str = request.headers.get("Authorization", None)
+    if not auth:
+        return Error(
+            code=ErrorCode.INVALID_REQUEST, errors=["Authorization header is expected"]
+        )
+    parts = auth.split()
```

Comment:
If `auth` were somehow a nonempty but blank string, e.g. `"         "` or `"\t"`, then `"    ".split() == []`, which could cause `parts[0]` to throw an `IndexError`.

It's unlikely, and maybe already impossible, depending on how `request.headers` is constructed. That said, we can guarantee this will never happen if we bind `request.headers.get("Authorization", "").strip()` to `auth` on line 17.

Reply:
That's a non-issue, as the auth headers are validated by Azure Functions.  That said, I simplified the checks which perform what you're after.